agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/common/AggregatingLogger.java

```diff
@@ -177,6 +178,17 @@ public void run() {
                 });
         logger.warn(message.toString());
       }
+
+      if (numSuccesses != 0) {
```

Comment:
ignore this file. i will revert it. push by mistake. logging success might be too much noise.